.github/workflows/autobuild.yml

```
@@ -45,29 +42,25 @@ jobs:
 
 
      steps:
-         # Checkout code
          - name:                    Checkout code
            uses:                    actions/checkout@v2
 
-         # Set variables
-         # Determine release / pre-release
          - name:                    Get Jamulus build info, determine actions & variables
```

```suggestion
         - name:                    Get Jamulus build/release info, determine actions & variables
```

Although not part of this PR, I don't like this description at all. What is "determine actions"?

---

```suggestion
         - name:                    Determine release version, type and prerelease variables
```

At least, that's what the script does. I'm open to other wordings though. :)

---

Yes. That's better.